src/main/java/com/google/step/finscholar/data/ServletConstantValues.java

```diff
@@ -23,13 +23,29 @@
     // The default value for undefined fields.
     public static final String DEFAULT_VALUE = "";
 
-    // Constants for exceptions when retrieving data from the Firestore database.
+    // Constants for the different Firestore Collection names.
+    public static final String SCHOLARSHIP_COLLECTION_NAME = "scholarships";
+    public static final String COLLEGE_COLLECTION_NAME = "colleges";
+    public static final String FORUM_POST_COLLECTION_NAME = "forumPosts";
+    public static final String FORUM_COMMENT_COLLECTION_NAME = "forumCommments";
+
+    // Constants for exceptions when retrieving data from the Firestore Database.
+    public static final String UNABLE_TO_WRITE_TO_FIRESTORE = "Unable to write new object to collection with name: ";
+    public static final String UNABLE_TO_READ_FROM_FIRESTORE = "Unable to retrieve data from Firestore.";
+    public static final String DOCUMENT = "Document with ID: ";
+    public static final String NEW_DOCUMENT_ADDED = "New Document added to: ";
+    public static final String AT = " at: ";
+    public static final String DNE = " does not exist.";
     public static final String UNABLE_TO_LOAD_FIREBASE = "Unable to load firebase: ";
 
-    // Cosntants for FirebaseAppManager.
+    // Constants for FirebaseAppManager.
     public static final String SCOPE_CLOUD_URL = "https://www.googleapis.com/auth/cloud-platform";
     public static final String SCOPE_USER_INFO_URL = "https://www.googleapis.com/auth/userinfo.email";
     public static final String DATABASE_URL = "https://viewing-step-2020-v2.firebaseio.com";
     public static final String PROJECT_ID = "viewing-step-2020-v2";
 
+    // Constants for FirebaseStorageManagerTest.
+    public static final String TEST_COLLECTION_NAME = "testObjects";
+    public static final String TEST_DOCUMENT_NAME = "testDocument";
+    public static final String EXPECTED_DOCUMENT_RETRIEVABLE = "{\"one\":\"testObjects\",\"two\":\"testDocument\"}";
```

Comment:
Don't pollute app code with test code. Make a new file in the test directory for test constants.